var/spack/repos/builtin/packages/pfft/package.py

```diff
@@ -27,46 +27,50 @@ def configure(self, spec, prefix):
 
         configure = Executable('../configure')
 
-        if '+double' in spec['fftw']:
+        fftw_precisions = spec['fftw'].variants.get('precision')
+        if 'double' in fftw_precisions:
             with working_dir('double', create=True):
                 configure(*options)
-        if '+float' in spec['fftw']:
+        if 'float' in fftw_precisions:
```

Comment:
The idiomatic way to do checks like this is:
```python
fftw = spec['fftw']
if 'precision=float' in fftw:
    ...
```
Can you change the PR accordingly?

Reply:
Sorry, python is not my "native tongue".   While the original version worked, I agree suggestion from alalazo is clearer, and have modified.